modules/openapi-generator/src/main/resources/swift5/XcodeGen.mustache

```diff
@@ -4,7 +4,7 @@ targets:
     type: framework
     platform: iOS
     deploymentTarget: "9.0"
-    sources: [{{projectName}}]
+    sources: [{{#swiftPackagePath}}{{swiftPackagePath}}{{/swiftPackagePath}}{{^swiftPackagePath}}{{#useSPMFileStructure}}Sources/{{projectName}}{{/useSPMFileStructure}}{{^useSPMFileStructure}}{{projectName}}/Classes{{/useSPMFileStructure}}{{/swiftPackagePath}}]
```

Comment:
@aymanbagabas Here the logic is a bit different, XcodeGen only need's the root directory.
If should be `swiftPackagePath` or `Sources` or `projectName`

```
sources: [{{#swiftPackagePath}}{{swiftPackagePath}}{{/swiftPackagePath}}{{^swiftPackagePath}}{{#useSPMFileStructure}}Sources{{/useSPMFileStructure}}{{^useSPMFileStructure}}{{projectName}}{{/useSPMFileStructure}}{{/swiftPackagePath}}]
```

Reply:
Note that `swiftPackagePath` won't be the root directory, it would be the full path to the source files.

Reply:
Yes, I know.
It will work as well.
This change is more to keep the current behaviour for the existing `XcodeGen.`

Reply:
Changes pushed